src/main/java/umc/spring/web/controller/StoreController.java

```diff
@@ -0,0 +1,68 @@
+package umc.spring.web.controller;
+
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.Parameters;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+import umc.spring.Vaildator.annotation.CheckPage;
+import umc.spring.apiPayload.ApiResponse;
+import umc.spring.repository.ReviewRepository.ReviewRepository;
+import umc.spring.service.ReviewService.ReviewQueryServiceImpl;
+import umc.spring.service.StoreService.StoreQueryServiceImpl;
+import umc.spring.web.controller.dto.ReviewDTO.ReviewRequestDTO;
+import umc.spring.web.controller.dto.ReviewDTO.ReviewResponseDTO;
+import umc.spring.web.controller.dto.StoreDTO.StoreRequestDTO;
+import umc.spring.web.controller.dto.StoreDTO.StoreResponseDTO;
+
+@RestController
+@RequiredArgsConstructor
+@Validated
+@RequestMapping("/stores")
+public class StoreController {
+
+    private final StoreQueryServiceImpl storeQueryService;
+
+
+    @PostMapping("{regionId}")
+    public ApiResponse<StoreResponseDTO.JoinResultDTO>
+    join(
+            @PathVariable Long regionId,
+            @RequestBody @Valid StoreRequestDTO.joinStoreDTO request
+    ) {
```

Comment:
코드 일관성을 위해 줄 바꿈 형태를 통일하는게 좋을 거 같아요!!
```suggestion
    public ApiResponse<StoreResponseDTO.JoinResultDTO> join(
            @PathVariable Long regionId,
            @RequestBody @Valid StoreRequestDTO.joinStoreDTO request
    ) {
```